package.json

```diff
@@ -9,17 +9,19 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "next": "11.0.1",
+    "next": "^12.0.4",
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "styled-components": "^5.3.0"
   },
   "devDependencies": {
-    "@types/styled-components": "^5.1.11",
     "@types/react": "17.0.14",
+    "@types/styled-components": "^5.1.11",
+    "@typescript-eslint/parser": "^3.0.0",
     "eslint": "7.30.0",
     "eslint-config-next": "11.0.1",
     "prettier": "^2.3.2",
+    "prettier-eslint": "10.1.0",
```

Comment:
The dependencies added here were an attempt to get prettier eslint to work. I haven't gotten it totally working yet.